page-030.tex

```diff
@@ -48,7 +48,7 @@
 
 It sometimes happens that one has passed with the point, and thrust
 that the Enemy in that moment pulls his Arm back and [laederar???]; In
-this it the [faut???] of the one who passed, taht he did not Continued
+this it the [faut???] of the one who passed, that he did not Continued
```

Comment:
Re: `faut`: faute means "mistake" in French.

Reply:
Go for it.